test/model/store_test.dart

```diff
@@ -652,6 +655,90 @@ void main() {
     test('retries on MalformedServerResponseException', () {
       checkRetry(() => connection.prepare(httpStatus: 200, body: 'nonsense'));
     });
+
+    group('reportErrorToUserBriefly', () {
```

Comment:
nit:
```suggestion
    group('error reporting', () {
```

Otherwise it sounds like these are tests of `reportErrorToUserBriefly`.